api/apps/curations/model.coffee

```diff
@@ -0,0 +1,80 @@
+#
+# Library of retrieval, persistance, validation, json view, and domain logic
+# for the "curation" resource.
+#
+
+_ = require 'underscore'
+_s = require 'underscore.string'
+db = require '../../lib/db'
+User = require '../users/model'
+async = require 'async'
+Joi = require 'joi'
+Joi.objectId = require('joi-objectid') Joi
+moment = require 'moment'
+request = require 'superagent'
+{ ObjectId } = require 'mongojs'
+{ ARTSY_URL, API_MAX, API_PAGE_SIZE } = process.env
+
+OPTIONS = { allowUnknown: true, stripUnknown: false }
+
+#
+# Schemas
+#
+schema = (->
+  id: @objectId()
+  name: @string().allow('', null)
+).call Joi
+
+querySchema = (->
+  limit: @number().max(Number API_MAX).default(Number API_PAGE_SIZE)
+  offset: @number()
+).call Joi
+
+#
+# Retrieval
+#
+@find = (id, callback) ->
+  query = if ObjectId.isValid(id) then { _id: ObjectId(id) }
+  db.curations.findOne query, callback
+
+@where = (input, callback) ->
+  Joi.validate input, querySchema, OPTIONS, (err, input) =>
```

Comment:
I wonder how useful querying across curations will be. Just want to note that `OPTIONS` here will allow any query to Mongo's `find` through. Which is fine b/c this is admin only, but maybe preferable to leave off this `validate` call out of the gate.